.dockstore.yml

```diff
@@ -60,6 +60,11 @@ workflows:
      testParameterFiles:
        - /cwl-training/exercise3/hello_examples/HelloGoodbye.json
      name: HelloGoodbye_cwl
+     - subclass: CWL
```

Comment:
Indentation for this line is wrong; it needs to be aligned with line 58.

Reply:
Fixed it!